sources/packages/web/Dockerfile

```diff
@@ -1,15 +1,21 @@
-FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 as builder
+FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 AS builder
 
 
 COPY ./package*.json ./
 RUN npm ci
 COPY . ./
 RUN npm run build
 
-FROM registry.access.redhat.com/ubi8/nodejs-16:1-5 as deployer
+FROM registry.access.redhat.com/ubi8/nginx-120 AS deployer
 
-RUN npm i -g serve
-COPY --from=builder /opt/app-root/src/dist .
-COPY --from=builder /opt/app-root/src/serve.json .
+RUN mkdir /app
 
-CMD ["sh", "-c","serve", "-p", "$PORT", "-s", "."]
\ No newline at end of file
+# Copy the app built on build-stage to the resulting container.
+COPY --from=builder /opt/app-root/src/dist /app
+
+# Copying the configuration file.
+COPY nginx.conf /etc/nginx/nginx.conf
+
+# Daemon off makes nginx to run on the foreground with only one process.
+# Docker will kill the container if the process dies.
+CMD ["nginx", "-g", "daemon off;"]
```

Comment:
Should we use `daemon off` in production?

Reply:
Anywhere we have a docker image. That way (with `daemon off`) if the process dies, the container is destroyed and a new container starts.